src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -244,9 +244,19 @@ const QuickMarcEditor = ({
     if (!skipValidation) {
       const closeValidationModal = manageBackendValidationModal();
 
-      newValidationErrors = await validate(getState().values);
+      try {
+        newValidationErrors = await validate(getState().values);
```

Comment:
Is this just for a failed HTTP request?

Reply:
This is for all possible errors, if we don't do this, any potential issue in the `validate` function will cause the validation modal to hang.